bindings/netstandard/ElectionGuard/ElectionGuard.Encryption/ExceptionHandler.cs

```diff
@@ -0,0 +1,16 @@
+﻿namespace ElectionGuard
+{
+    public static class ExceptionHandler
+    {
+        public static void GetData(out string function, out string message, out ulong code)
+        {
+            var status = NativeInterface.ExceptionHandler.GetData(out var functionData, out var messageData, out var codeData);
+            function = functionData.PtrToStringUTF8();

```

Comment:
in `StatusExtensions` is the extension `ThrowIfError` but maybe we should create a `ReturnIfError` in the event we get a `ELECTIONGUARD_STATUS_ERROR_BAD_ALLOC` back?

Reply:
Added getting the data in the extension for now when there is an error.